### PR TITLE
11.0 mig l10n ch pain credit transfer

### PR DESCRIPTION
**Pull request by mpanarin**
migration of l10n_ch_pain_credit_transfer to odoo 11

N.B.
---------
it requires l10n_ch_pain_base
- [x] https://github.com/OCA/l10n-switzerland/pull/384

### CODE REVIEW

**Comment by simahawk** [comment]
@mpanarin check travis and runbot pls

**Comment by simahawk** [comment]
also, squash translations commits all in one

**Comment by yvaucher** [comment]
@mpanarin can you also remove the file `l10n_ch_pain_credit_transfer/i18n/nb_NO.po` while squashing the translation commits ?

File: l10n_ch_pain_credit_transfer/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+In the menu *Accounting > Configuration > Management > Payment Methods*,
+select the payment method that has the code *sepa_credit_transfer* and
+set the *PAIN Version* to *pain.001.001.03.ch.02 (credit transfer in Switzerland)*.
+
+Usage
+=====
+
+On the payment order, you will see a new computed boolean field named
+*BVR* which shows if the payment order is BVR or not.
```
Comment (yvaucher) [inline l10n_ch_pain_credit_transfer/README.rst]: Please rename BVR by ISR which is the english term

**Review by yvaucher — CHANGES_REQUESTED** [review]
Only a small change in README

File: l10n_ch_pain_credit_transfer/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# copyright 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Switzerland - PAIN Credit Transfer",
+    "summary": "Generate ISO 20022 credit transfert (SEPA and not SEPA)",
+    "version": "11.0.1.0.0",
+    "category": "Finance",
+    "author": "Akretion,Camptocamp,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
```
Comment (simahawk) [inline l10n_ch_pain_credit_transfer/__manifest__.py:10]: website: https://github.com/OCA/l10n-switzerland

**Review by simahawk — CHANGES_REQUESTED** [review]
@mpanarin add `oca_dependecies.txt` to include the other PR. See https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/oca_dependencies.txt

**Review by grindtildeath — APPROVED** [review]
Code review only 👍

**Comment by grindtildeath** [comment]
BTW please fix tests

**Comment by asaunier** [comment]
Suggested changes seem to have been done. @yvaucher @simahawk do you approve the PR now?
Required PR #384 seems OK as well.

Concerning the failing tests, I think I have seen the same errors with ``test_sct_ch_payment_type1`` and ``test_sct_ch_payment_type3`` in some other PRs. Aren't those tests broken in the 11.0 branch?

**Review by yvaucher — APPROVED** [review]

**Comment by yvaucher** [comment]
@mpanarin there is a conflict on oca_dependencies.txt, can you check?

**Comment by yvaucher** [comment]
travis seems unhappy with the tests

**Comment by mpanarin** [comment]
@yvaucher requires this fix https://github.com/OCA/l10n-switzerland/pull/417

**Comment by yvaucher** [comment]
#417 merged

**Pull request merged**